worlds/lingo/__init__.py

```diff
@@ -89,10 +89,23 @@ def create_items(self):
                     pool.append(self.create_item("Puzzle Skip"))
 
             if traps:
-                traps_list = ["Slowness Trap", "Iceland Trap", "Atbash Trap"]
-
-                for i in range(0, traps):
-                    pool.append(self.create_item(traps_list[i % len(traps_list)]))
+                trap_weights = {name: self.options.trap_weights.get(name, 0) for name in TRAP_ITEMS}
```

Comment:
Not really a problem, just pointing out that you're using TRAP_ITEMS to generate the option schema, then using TRAP_ITEMS to generate this dict, yet still using `.get(name, 0)` - Since your Schema doesn't list any of the keys as "optional", all of these keys will always exist, so you could also just `[name]`, at which point you can reduce it further to just use the option dict directly

That's what I did with some of my recent commits on my trap weights PR

Reply:
Done.